src/test/script_tests.cpp

```diff
@@ -2623,12 +2623,170 @@ BOOST_AUTO_TEST_CASE(IsWitnessProgram)
     }
 }
 
+
+class AlwaysGoodSignatureChecker : public BaseSignatureChecker
+{
```

Comment:
Maybe these tests make more sense to be in src/sigopcount_tests.cpp ?

Reply:
yes